server/src/main/java/org/elasticsearch/search/aggregations/bucket/terms/TermsAggregatorFactory.java

```diff
@@ -130,6 +134,7 @@ public Aggregator build(
                 }
 
                 // TODO: [Zach] we might want refactor and remove ExecutionMode#create(), moving that logic outside the enum
+                logger.info("Creating bytes terms aggregator with execution mode [" + execution + "]");
```

Comment:
These'd probably be fine to keep with `debug` if you like`. Though you should do the `[{}]` substitution thing.